boards/phynode-kw41z/Kconfig

```diff
@@ -13,5 +13,8 @@ config BOARD_PHYNODE_KW41Z
     select BOARD_COMMON_KW41Z
     select HAS_PERIPH_I2C
     select HAS_PERIPH_SPI
+    select HAVE_CCS811
```

Comment:
not a big deal but is seems there is a convention to have a newline to separate `HAS_` and `HAVE_`

Reply:
oops, wanted to delete this one... It is super minor.

Reply:
Thanks, I fixed this.